frontend/assets/style/components/_forms.scss

```diff
@@ -129,10 +129,6 @@ textarea,
     border-right: 1px solid $general_0;
   }
 
-  * + .input {
-    border: 0;
```

Comment:
In which scenarios this can happen/be helpful?
It would be nice to leave a comment in the code explaining that since this selector is global...

Reply:
Sorry, I'm not sure to follow..

Reply:
Nevermind, when I saw it I thought you added it, not removed it. Pardon the confusion